policy/release/lib/attestations.rego

```diff
@@ -51,16 +51,14 @@ pipelinerun_attestations := att if {
 
 pipelinerun_slsa_provenance02 := [statement |
 	some att in input.attestations
-	statement := lib.statement(att)
-	statement.predicate.buildType in pipelinerun_att_build_types
+	att.statement.predicate.buildType in pipelinerun_att_build_types
```

Comment:
Re the "not safe" errors, `statement` on line line 52 means use the `statement` var from the following block. So you have to make sure it's set. Something like this might do it:

```suggestion
	statement := att.statement
	statement.predicate.buildType in pipelinerun_att_build_types
```

Reply:
It's a similar issue for the the others iiuc.